Bulldozer.CSV/Maps/Benevolence.cs

```diff
@@ -228,10 +228,22 @@ private int LoadBenevolenceRequest( CSVInstance csvData )
                     if ( string.IsNullOrWhiteSpace( benevolenceRequestRequestedById ) )
                     {
                         // Handle Address
-                        var requestAddress = new LocationService( lookupContext ).Get( benevolenceRequestAddress.Left( 100 ), benevolenceRequestAddress2.Left( 100 ), benevolenceRequestCity, benevolenceRequestState, benevolenceRequestZip, benevolenceRequestCountry, verifyLocation: false );
-                        if ( requestAddress != null )
+
+                        if ( !string.IsNullOrWhiteSpace( benevolenceRequestAddress ) || !string.IsNullOrWhiteSpace( benevolenceRequestCity ) || !string.IsNullOrWhiteSpace( benevolenceRequestState ) )
```

Comment:
@guymaness2 Are these if statements necessary? I thought before you were trying to stand it up regardless of if they were empty or not and then let the logger handle saying what is wrong. Because now you will not get a log entry if all 3 are empty

Reply:
@nateh777 Correct. This was intentionally added to increase performance by bypassing any records with no address provided at all. Without it, all records with no address being imported throws an exception. This is due to the way I am defaulting country to "US" if it is missing. I can either remove the country default logic altogether which will then make the IF wrapper unnecessary or I can move the IF wrapper into the GetOrAddLocation method around the country logic to only set an empty country in this scenario. Let me know which you would prefer.